### PR TITLE
docs(Tooltip): add docs for animate_position prop

### PR DESCRIPTION
**Pull request by langz**
Saw we had an example using the prop, `animate_position`, in https://eufemia.dnb.no/uilib/components/tooltip/#tooltip-with-position-animation
Thought it could be nice to add the property to the list of properties, and a small explanation of what it is.

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/tooltip/properties.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ showTabs: true
 | `position`                                  | _(optional)_ defines the offset position to the target element the arrow appears. Can be `top`, `right`, `left` and `bottom`. Defaults to `top`. |
 | `align`                                     | _(optional)_ defines the offset alignment to the target element the arrow appears. Can be `center`, `right` and `left`.Defaults to `center`.     |
 | `arrow`                                     | _(optional)_ defines the direction where the arrow appears. Can be `center`, `top`, `right`, `bottom` and `left`. Defaults to `center`.          |
+| `animate_position`                          | _(optional)_ I'm not sure what to write here, @tujoworker?                                                                                       |
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/tooltip/properties.md:15]: @tujoworker, if you could assist with a line here, I can update the PR and merge it, etc...

Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/components/tooltip/properties.md:15]:
```suggestion
| `animate_position`                          | _(optional)_ set to `true` to animate a single Tooltip from one element to another element. You also need to set a unique `group` name.                                                                                       |
```

Something like this?

**Review by tujoworker — APPROVED** [review]
Yeah, nice 🙏

File: packages/dnb-design-system-portal/src/docs/uilib/components/tooltip/properties.md (hunk 1 above)
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/components/tooltip/properties.md]: Perfecto 🇮🇹

**Pull request merged**